schemas/dependencies/jira-board-1.yml

```diff
@@ -31,6 +31,13 @@ properties:
         type: string
     required:
     - workspace
+  issueType:
+    type: string
+    enum:
+    - Task
+    - Bug
+    - Ticket
+    - Incident
```

Comment:
i understand the idea of having a sanity check for issue types at the schema level. afaik issue types are jira instance and project specific (e.g. ticket and incident is not something i can use in our team project). hence we can't rely on this to make sure an alert will be created successfully on a specific board. 

i suggest to remove to enum

Reply:
Makes sense. Will drop it.

Reply:
Done.

Reply:
thanks